recipes/libyuv/all/patches/1892-0001-fix-cmake.patch

```diff
@@ -10,7 +12,16 @@
  OPTION( UNIT_TEST "Built unit tests" OFF )
  
  SET ( ly_base_dir	${PROJECT_SOURCE_DIR} )
-@@ -139,15 +139,10 @@ int main(void) { return 0; }
+@@ -74,8 +74,6 @@ if(MSVC)
+   ADD_DEFINITIONS ( -D_CRT_SECURE_NO_WARNINGS )
+ endif()
+ 
+-# Need to set PIC to allow creating shared libraries from object file libraries.
+-SET(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
This is removed so static builds can chose fpic. For shared ones, ON is already set by Conan

Reply:
Hmm, my intuition/expectations seem to be wrong, Linux fails without explicitly settings this to True